futures-channel/src/lib.rs

```diff
@@ -12,11 +12,6 @@
 
 #![doc(html_root_url = "https://rust-lang-nursery.github.io/futures-doc/0.3.0-alpha.2/futures_channel")]
 
-#[cfg(feature = "std")]
```

Comment:
Oh, I didn't think about that. We'll have to take care not to accidentally use `std` symbols in our `no_std` build, because there won't be anything preventing them in CI (since `std` is still reachable via a path).

Reply:
I'm pretty sure that this will catch them:

https://github.com/rust-lang-nursery/futures-rs/blob/3fdcfc1c42fd507b3e04c6ed2e99dd7e5cf1b56e/.travis.yml#L50

Reply:
Yeah, that'd get it. I opened https://github.com/rust-lang/rust/issues/53166 to discuss this.